### PR TITLE
Adjust all graphs to default to 'All' timeline

### PR DESCRIPTION
**Pull request by vincenthongzy**
https://www.notion.so/nethermind/Default-graph-time-range-to-All-for-more-visually-different-graph-8bf8940b6a7448efb6b14c4081d82977?pvs=4

### CODE REVIEW

**Review by Zishan-7 — APPROVED** [review]

**Review by rubo — CHANGES_REQUESTED** [review]
- Refactored LRT/LST approach to have a single point of control. Using `ParentSize` is problematic as it fires lots of negative values.
- All other changes have no effect as setting the default selected key doesn't fire the handler. AVS TVL is a fixed sample suggestion.

**Comment by rubo** [comment]
I'm not sure changing the default time range is a good idea at all. The more time passes, the less sense it makes.

File: src/avs/charts/OperatorsTabLineChart.jsx [hunk 1]
```diff
@@ -264,7 +268,8 @@ export default function OperatorsTabLineChart({ points, height, width }) {
   );
 }
 
-const margin = { top: 20, right: 40, bottom: 24, left: 0 };
+const margin = { top: 20, right: 40, bottom: 30, left: 20 };
```
Comment (vincenthongzy) [inline src/avs/charts/OperatorsTabLineChart.jsx:271]: bumped the margins for line charts, otherwise the leftmost date label on 1w graphs gets gobbled up

Comment (vincenthongzy) [reply src/avs/charts/OperatorsTabLineChart.jsx:271]: image

Notice `Aug` also gets cut off

**Review by Zishan-7 — APPROVED** [review]

File: src/operators/charts/OperatorRestakersLineChart.jsx [hunk 2]
```diff
@@ -342,7 +346,8 @@ function LineChart({ points, height, width }) {
   );
 }
 
-const margin = { top: 20, right: 40, bottom: 24, left: 0 };
+const margin = { top: 20, right: 40, bottom: 30, left: 20 };
```
Comment (rubo) [inline src/operators/charts/OperatorRestakersLineChart.jsx:349]:
```suggestion
const margin = { top: 20, right: 40, bottom: 30, left: 0 };
```

File: src/operators/charts/OperatorTVLLineChart.jsx [hunk 3]
```diff
@@ -373,7 +377,8 @@ function LineChart({ points, height, width }) {
   );
 }
 
-const margin = { top: 20, right: 40, bottom: 24, left: 0 };
+const margin = { top: 20, right: 40, bottom: 30, left: 20 };
```
Comment (rubo) [inline src/operators/charts/OperatorTVLLineChart.jsx:380]:
```suggestion
const margin = { top: 20, right: 40, bottom: 30, left: 0 };
```

**Pull request merged**